src/Umbraco.TestData/SegmentTestController.cs

```diff
@@ -50,7 +50,7 @@ public ActionResult DisableDocTypeSegments(string alias)
             if (!ct.VariesBySegment())
                 return Content($"The document type {alias} does not allow segments, nothing has been changed");
 
-            ct.Variations = ct.Variations.UnsetFlag(ContentVariation.Segment);
+            ct.Variations = ct.Variations.SetVariesBy(ContentVariation.Nothing);
```

Comment:
@ronaldbarendse Alright I've fixed the build now, I had to also update this file. This particular change I am not 100% sure about, in this method we're trying to set the variation to `Nothing`, is that correct?

And general note, it would be good if you could compile before committing next time.

Reply:
`Nothing` is not a flag, because the value is `0` (this is the same as expecting `1 - 0 = 0`, where 1 is the current flag and 0 the flag you're trying to remove). Nonetheless, it should not set the variation to `Nothing`, but only remove the `Segment` flag. I've updated this PR to do exactly this 👍